<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_demoClass</class>
 <widget class="QMainWindow" name="qt_demoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>基于协议分析的网络取证工具</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>601</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelDev">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>网卡接口:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonArp">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ARP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IPv4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUdp">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTcp">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIcmp">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ICMP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonHttp">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HTTP</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetArp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>901</width>
      <height>461</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>硬件类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>协议类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>硬件长度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>协议长度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>op</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源MAC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的MAC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的ip</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetIp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>901</width>
      <height>461</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>源ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>服务类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ip报文长度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>标识</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TTL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>协议</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>首部检验和</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetUdp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>901</width>
      <height>461</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <column>
     <property name="text">
      <string>源ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>udp报文长度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>udp检验和</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetTcp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>901</width>
      <height>461</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>源ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>seq</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ack</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>窗口大小</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>tcp检验和</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>紧急指针</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetHttp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>901</width>
      <height>461</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>Http请求头部</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>源端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>表单数据</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidgetIcmp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>901</width>
      <height>461</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>源ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>目的ip</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>代码</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelProtoType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>协议类型:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPacket">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>已获取数据包:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPacketNum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>comboBox</zorder>
   <zorder>labelDev</zorder>
   <zorder>pushButtonStart</zorder>
   <zorder>pushButtonStop</zorder>
   <zorder>pushButtonArp</zorder>
   <zorder>pushButtonIp</zorder>
   <zorder>pushButtonUdp</zorder>
   <zorder>pushButtonTcp</zorder>
   <zorder>pushButtonIcmp</zorder>
   <zorder>pushButtonHttp</zorder>
   <zorder>labelStatus</zorder>
   <zorder>labelProtoType</zorder>
   <zorder>labelPacket</zorder>
   <zorder>labelPacketNum</zorder>
   <zorder>tableWidgetArp</zorder>
   <zorder>tableWidgetHttp</zorder>
   <zorder>tableWidgetIcmp</zorder>
   <zorder>tableWidgetIp</zorder>
   <zorder>tableWidgetUdp</zorder>
   <zorder>tableWidgetTcp</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt_demo.qrc"/>
 </resources>
 <connections/>
</ui>
